server/src/main/java/com/defold/extender/TreePrinter.java

```diff
@@ -0,0 +1,59 @@
+package com.defold.extender;
```

Comment:
This was useful while testing to quickly get an overview of the folder structure when processing or building android packages. We can remove it and I'll put it in a gist for future use.

Reply:
I think it's ok to have here.
Fyi, I generally use the `tree` command on the command line.
(I.e. not deleting the job folder, then logging into the server)